capstone/capapi/routers.py

```diff
@@ -0,0 +1,44 @@
+class CAPAPIRouter(object):
+    """
+    Determine how to route database calls for an app's models (in this case, for an app named capapi).
+    All other models will be routed to the next router in the DATABASE_ROUTERS setting if applicable,
+    or otherwise to the default database.
+    """
+
+    def db_for_read(self, model, **hints):
+        """Send all read operations on capapi app models to `capapi`."""
+        if model._meta.app_label == 'capapi':
+            return 'capapi'
+        return None
+
+    def db_for_write(self, model, **hints):
+        """Send all write operations on capapi app models to `capapi`."""
+        if model._meta.app_label == 'capapi':
+            return 'capapi'
+        return None
+
+    def allow_relation(self, obj1, obj2, **hints):
+        """Determine if relationship is allowed between two objects."""
+        # Allow any relation between two models that are both in the capapi app.
+        if obj1._meta.app_label == 'capapi' and obj2._meta.app_label == 'capapi':
+            return True
+        # No opinion if neither object is in the capapi app (defer to default or other routers).
+        elif 'capapi' not in [obj1._meta.app_label, obj2._meta.app_label]:
+            return None
+
+        # Block relationship if one object is in the capapi app and the other isn't.
+            return False
```

Comment:
Extra indent breaks the logic here.

(Also style nitpick: maybe reverse the elif test so all of these functions end with `return None`?)